docs/Technical_Reference.html

```diff
@@ -69,6 +69,7 @@
 <ul>
 <li><a class="el" href="tr_FeatureExtraction.html">Feature Extraction</a></li>
 <li><a class="el" href="tr_integration.html">Integrating your Application in CaPTk</a></li>
+<li><a class="el" href="tr_integration.html#tr_sourceBuid">Building CaPTk from source</a></li>
```

Comment:
Do we need to modify the *.html files for doc update if we have updated the *.txt file?

Reply:
Yes, because the documentation in https://cbica.github.io/CaPTk/ is picked up from the ```docs``` folder and that is updated with ```CAPTK_UPDATE_DOCS``` is enabled. Since we need the HTML files for packaging in the application itself, this ensured that we didn't have to rely on the same mechanism for the online documentation as well.